<?xml version="1.0"?>

<launch> 

<group ns="/robot_0/map_merge">
  <param name="init_pose_x" value="0.0" />
  <param name="init_pose_y" value="0.0" />
  <param name="init_pose_z" value="0.0" />
  <param name="init_pose_yaw" value="0.0" />
</group>

<group ns="/robot_1/map_merge">
  <param name="init_pose_x" value="-0.0" />
  <param name="init_pose_y" value="-0.8" />
  <param name="init_pose_z" value="0.0" />
  <param name="init_pose_yaw" value="0.0" />
</group>

<group ns="/robot_2/map_merge">
  <param name="init_pose_x" value="-0.0" />
  <param name="init_pose_y" value="0.8" />
  <param name="init_pose_z" value="0.0" />
  <param name="init_pose_yaw" value="0.0" />
</group>

</launch>
